makeflow/src/makeflow.c

```diff
@@ -76,6 +76,11 @@ an example.
 
 #define MAX_REMOTE_JOBS_DEFAULT 100
 
+struct {
```

Comment:
@btovar suggestion, you could actually use an rmsummary struct in place of this newly created struct as it has fields for these values.